soap/src/java/com/zimbra/soap/admin/message/ModifyVolumeRequest.java

```diff
@@ -32,32 +32,23 @@
 @XmlRootElement(name=AdminConstants.E_MODIFY_VOLUME_REQUEST)
 public class ModifyVolumeRequest {
 
-    /**
-     * @zm-api-field-tag volume-id
-     * @zm-api-field-description Volume ID
-     */
-    @XmlAttribute(name=AdminConstants.A_ID, required=true)
-    private short id;
-
     /**
      * @zm-api-field-description Volume information
      */
     @XmlElement(name=AdminConstants.E_VOLUME, required=true)
-    private VolumeInfo volume;
+    private VolumeInfo volumeInfo;
 
     /**
      * no-argument constructor wanted by JAXB
      */
      @SuppressWarnings("unused")
     private ModifyVolumeRequest() {
-        this((short)-1, (VolumeInfo)null);
+        this((VolumeInfo)null);
```

Comment:
Please revert constructor changes, please do not update existing contracts. We don't have enough time to find all references and fix them everywhere.